fees/src/lib.rs

```diff
@@ -1,13 +1,14 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
-use near_sdk::env::state_exists;
 use near_sdk::json_types::{U128, U64};
 use near_sdk::{env, near_bindgen, AccountId, PanicOnDefault};
 use std::error::Error;
 use std::fmt::{Display, Formatter};
 use std::num::ParseFloatError;
 use std::str::FromStr;
 
-// We multiply the percent to 100 here to get rid of the floating numbers.
+// We multiply percents to 100 here to get rid of the floating numbers.
+const MIN_FEE_PERCENT: u64 = 1; // 0.01 %
+const MAX_FEE_PERCENT: u64 = 1000; // 10 %
 const DEFAULT_PERCENT: U64 = U64(500); // 5%
```

Comment:
Probably we can make MIN/MAX FEE constants U64 type too. This way in the code:

```
    if result < MIN_FEE_PERCENT {
        Err(ParseError::TooLowPercent)
    } else if result > MAX_FEE_PERCENT {
        Err(ParseError::TooHighPercent)
    } else {
        Ok(U64(result))
    }
```

We can wrap result in U64 immediately and use this comparison:

```
    if result < MIN_FEE_PERCENT {
        Err(ParseError::TooLowPercent)
    } else if result > MAX_FEE_PERCENT {
        Err(ParseError::TooHighPercent)
    } else {
        Ok(result)
    }
```

Reply:
But in this case U64 type would require to support Eq, PartialEq 🤷

Reply:
Yes. That's why I don't wrap the `MIN_FEE_PERCENT` and `MAX_FEE_PERCENT` in the `U64`. And implementation of the traits `Eq` and `PartialEq` on our side is overkill 😄

Reply:
Yeah, it works for me this way